src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,7 +1,745 @@
+
+// 할인 정보
+const DiscountInfoWrap = styled.div`
+  height: 120px;
+  border-bottom: 0.5px solid #f8f8f8;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  /* background-color: yellow; */
+  /* padding: 8px 0; */
+`;
+
+const DiscountInfo = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+const DiscountAmount = styled(MonthPrice)`
+  color: #fd3049;
+`;
+
+const CouponButton = styled.button`
+  display: flex;
+  justify-content: center;
+  width: 95%;
+  color: white;
+  font-weight: 500;
+  background-color: #fd3049;
+  outline: none;
+  border: none;
+  padding: 12px;
+  border-radius: 4px;
+  margin: 0 auto;
+`;
+
+// 상품 태그
+const ProductTag = styled.div`
+  height: 120px;
+  display: grid;
+  justify-items: start;
+  padding-left: 4px;
+  grid-template-columns: 1fr 1fr;
+`;
+
+const TagBox = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const TagIcon = styled.i`
+  margin-right: 4px;
+`;
+
+// 상품 구매
+const ProductBuy = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  height: 120px;
+`;
+
+const ShareBtnBox = styled.div`
+  display: flex;
+  justify-content: space-evenly;
+`;
+
+const ShareBtn = styled.button`
+  outline: none;
+  border: none;
+  padding: 12px;
+  font-weight: 500;
+`;
+
+const ShareBtnIcon = styled.i`
+  margin-right: 4px;
+`;
+
+const BuyButton = styled(CouponButton)`
+  background-color: rgb(255, 86, 0);
+  padding: 16px;
+  font-size: 16px;
+  font-weight: 700;
+`;
+// 사이드바 하단
+const SideBarBottom = styled.div`
+  width: 100%;
+  height: 75px;
+  /* background-color: greenyellow; */
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 0.5px solid lightgray;
+  -webkit-box-shadow: 3px 6px 14px -3px #e9e9e9;
+  box-shadow: 3px 6px 14px -3px #e9e9e9;
+  padding: 16px;
+`;
+
+const BannerTextWrap = styled.div`
+  width: 100%;
+  height: 74px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  line-height: 1.5;
+`;
+
+const BannerLogo = styled.div`
+  width: 35px;
+  height: 35px;
+  border-radius: 8px;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  background-color: black;
+  font-size: 12px;
+  font-weight: 700;
+`;
 
 function ProductDetail() {
-  return <div>ProductDetail</div>;
+  return (
+    <Wrapper>
+      <LeftSide>
+        <ThumbBox>
+          <ThumbImg src="../images/thumbImg.png" />
+          <ThumbCoupon>4만원 쿠폰</ThumbCoupon>
+        </ThumbBox>
+        <TabMenus>
+          <TabMenu color fontWeight border>
+            후기
+          </TabMenu>
+          <TabMenu>클래스 소개</TabMenu>
+          <TabMenu>커리큘럼</TabMenu>
+          <TabMenu>키트 소개</TabMenu>
+          <TabMenu>크리에이터</TabMenu>
+          <TabMenu>환불 정책</TabMenu>
+        </TabMenus>
+        <Banner bgColor="#77B053">
+          세대별 TOP 50 클래스 공개!
+          <br />
+          <p className="subText">인기 순위 보기 ></p>
```

Comment:
```suggestion
          <p className="subText">인기 순위 보기</p>
```
꺽쇠 때문에 에러가 뜨고 있을 것 같네요